clients/filesystem-fuse/src/fuse_api_handle.rs

```diff
@@ -30,10 +30,174 @@ use fuse3::{Errno, FileType, Inode, SetAttr, Timestamp};
 use futures_util::stream;
 use futures_util::stream::BoxStream;
 use futures_util::StreamExt;
+use log::debug;
 use std::ffi::{OsStr, OsString};
 use std::num::NonZeroU32;
 use std::time::{Duration, SystemTime};
 
+pub(crate) struct FuseApiHandleDebug<T: RawFileSystem> {
+    inner: FuseApiHandle<T>,
+}
+
+impl<T: RawFileSystem> FuseApiHandleDebug<T> {
+    pub fn new(fs: T, context: FileSystemContext) -> Self {
+        Self {
+            inner: FuseApiHandle::new(fs, context),
+        }
+    }
+
```

Comment:
`new` is needed, we can use it to create a `FuseApiHandleDebug`, and it will create a `FuseApiHandle` as inner file system.

e.g.
```
    let fs = DefaultRawFileSystem::new(gvfs);
    FuseApiHandleDebug::new(fs, fs_context)
```